components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java

```diff
@@ -16348,19 +16348,27 @@ public APIRevision getRevisionByRevisionUUID(String revisionUUID) throws APIMana
     /**
      * Get revision UUID providing revision number
      *
-     * @param revisionNum Revision number
-     * @param apiUUID     UUID of the API
+     * @param revisionNum   Revision number
+     * @param apiUUID       UUID of the API
+     * @param organization  organization ID of the API
      * @return UUID of the revision
      * @throws APIManagementException if an error occurs while retrieving revision details
      */
-    public String getRevisionUUID(String revisionNum, String apiUUID) throws APIManagementException {
+    public String getRevisionUUID(String revisionNum, String apiUUID, String organization) throws APIManagementException {
```

Comment:
can we have a new method for this

Reply:
ack

Reply:
Fixed with 7ebec1dc831d3a958a6721530f4d83ca1961ea07